.gitignore

```
@@ -47,3 +47,8 @@ node_modules/
 # Files generated by build script
 icons/*.js
 /index.js
+
+# Ignore all files in the icons folder
+icons/*
+# Except SVG files
+!*.svg
```

Shouldn't it be ⬇️ ?

```suggestion
!icons/*.svg
```

We might, in the future, want to ignore SVGs elsewhere

---

Totally, I already fixed it but did not commit 🤦‍♂️ 
Coming back to work after the holiday is not so easy 😅 
Should be better now, thanks!